### ISSUE TITLE
Import dump does not work if an index is empty

### ISSUE THREAD

**Issue by nicolasvienot** [meilisearch/MeiliSearch#1952]
**Describe the bug**

When trying to import a dump using `--import-dump` I have the following error:

```bash
➜  testown ./meilisearch-0.24.0 --master-key masterKey --import-dump dumps/20211130-202558793.dump
[2021-11-30T20:26:25Z INFO  meilisearch_lib::index_controller::dump_actor] Loading dump from 2021-11-30 20:25:58.794368 UTC, dump database version: 0.24.0, dump version: V3
[2021-11-30T20:26:25Z INFO  meilisearch_lib::index_controller::dump_actor::loaders::v3] Loading dump from 2021-11-30 20:25:58.794368 UTC, dump database version: 0.24.0, dump version: V3
Error: The `ndjson` payload must contain at least one document.
```

It seems that this is because I exported a dump that contains an index with 0 document. However, there is also an index with documents in this dump.

`/stats` from the MeiliSearch used to create the dump:
```json
{
    "databaseSize": 221184,
    "lastUpdate": "2021-11-30T20:30:50.946454Z",
    "indexes": {
        "movies": {
            "numberOfDocuments": 6,
            "isIndexing": false,
            "fieldDistribution": {
                "author": 6,
                "genre": 5,
                "id": 6,
                "price": 3,
                "title": 6
            }
        },
        "test": {
            "numberOfDocuments": 0,
            "isIndexing": false,
            "fieldDistribution": {}
        }
    }
}
```

**To Reproduce**

1. `./meilisearch-0.23.1 --master-key masterKey`
2. Create an index and add documents
3. Create an index and don't add documents
4. Create a dump (`/dumps`)
5. `./meilisearch-0.24.0 --master-key masterKey --import-dump theDumpExported`

**Expected behavior**
I should be able to import a dump from a MeiliSearch containing an index without documents

**MeiliSearch version:** 
Tested in those two contexts:
- Create a dump in `MeiliSearch 0.23.1` and import it in `MeiliSearch 0.24.0`
- Create a dump in `MeiliSearch 0.24` and import it in `MeiliSearch 0.24.0`

**Comment by MarinPostma** [comment]
Hey, we are already aware of this issue, and it is fixed as part of 0.25 :)

**Comment by nicolasvienot** [comment]
Perfect, thanks @MarinPostma 😄

**Comment by irevoire** [comment]
It was actually solved in 0.24. But it was a problem with the export, not the import, which means you'll need to somehow create your index with the 0.24
